learn.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+   <meta charset="UTF-8">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <script
+      src="https://kit.fontawesome.com/2d323a629b.js"
+      crossorigin="anonymous">
+   </script>
+   <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto&display=swap"
+   rel="stylesheet">
+   <link rel="stylesheet" href="css/tailwind_style.css">
+   <link rel="stylesheet" href="css/style.css">
+   <title>Learn | QC@UCI</title>
+   <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
+
+   <meta name="viewport" content="width=device-width, initial-scale=1">
+
+</head>
+<body>
+   <header class="bg-black text-white pt-2 pb-2 pl-8 pr-8>
+      <div id="left"><img src="image/QC@UCI Logo_small.png" width="125" alt="QC@UCI Logo"></div>
+      <nav>
+         <ul class="header__nav">
+            <li><a href="index.html">Home</a></li>
+            <li><a href="join.html">Join</a></li>
+            <li><a href="learn.html">Learn</a></li>
+            <li><a href="team.html">Team</a></li>
+            <li><a href="event.html">Events</a></li>
+            <li><a href="projects.html">Projects</a></li>
+            <li><a href="contact_us.html">Contact</a></li>
+         </ul>
+      </nav>
+   </header> 
+
+    <p style="margin-bottom:180px;"></p>
+
+   <div class="learn__title">What is Quantum Computing?</div>
+   <p style="margin-bottom:50px;"></p>
+
+   <div class="about-main-container">
+    <div class="about-left-section">
+        <p>
+         Quantum computing is a new paradigm of computing which uses physical quantum phenomena to drastically speed up certain computations. Unlike classical computers that rely on bits, which can be either 0 or 1, to process data, quantum computers use qubits which can be any value between 0 and 1, as well as complex values. This allows for much faster computations on problems such as factoring large numbers, which is a key part of modern encryption.
```

Comment:
Let's say "quantum physics" instead of "physical quantum phenomena". 

For the last sentence, you can say instead something like this "This allows for much faster computation due to increased parallelization and exponential speedup for specific applications such as factoring large ... <whatever you have> .

You can also bold the important terms like: 
- "quantum physics"
- "qubits"
- "parallelization"
- "exponential"

Reply:
Hey so I actually made a updated document with many changes could you comment on the google doc for any required changes:
https://docs.google.com/document/d/1GPkJ9DoUVZyNGQ2ii3jln-w3k6ElU8zqp0Lpe0jIEbs/edit

Reply:
I made another commit with updated information could you check that out.

Reply:
I also need images in the image folder labeled app-imageX.jpg X representing 1-5 numbers for each application category, could you help find them and maybe just add them into the images folder and the code should take care of everything else.